librocketchat.c

```diff
@@ -805,6 +837,11 @@ rc_fetch_url(RocketChatAccount *ya, const gchar *url, const gchar *postdata, Roc
 #if PURPLE_VERSION_CHECK(3, 0, 0)
 	
 	PurpleHttpRequest *request = purple_http_request_new(url);
+    if (ya->websocket_server != ya->server) {

```

Comment:
Does this need to be a string comparasion instead, eg `!purple_strequal(ya->websocket_server, ya->server)`?

Reply:
On my tests it did work without but might be better.